### ISSUE TITLE
Add callback for updating after the drag has ended

### ISSUE THREAD

**Issue by bzf** [AlexChekel1337/SliderControl#1]
Hi there 👋🏾 

Nice work on the library, I've been looking for an alternative for the default SwiftUI `Slider` view to use in my app and this looks _almost_ perfect!

My use-case is that I want the user to be able to drag the value around, and update the current duration label (like in your example image), but I'd only want to skip around the track _after_ the drag has ended.

What I'm looking for is some sort of `.onDragEnd` modifier that I could use to know that the value will stop updating so I have a good place to call into my playback code to do the actual skip.

Do you have any suggestions or workarounds I could use to figure it out?

Thanks! 🎉

**Comment by AlexChekel1337** [comment]
Hey, thanks for reaching out! 

I'm afraid current implementation does not allow for these callbacks to be added in any way. Normally I'd suggest changing the `isContinuous` property, but SwiftUI's `Slider` is always continuous, and for this reason `SliderControlView` also does not expose its `isContinuous` property, that is available with this control in UIKit.

SwiftUI's `Slider` has `onEditingChanged` callback right in its initializer, which seems to do exactly what you are trying to achieve. I'd like to continue following built-in controls' APIs, so I will implement the same callback for `SliderControlView` soon. It would look like this:
```swift
SliderControlView(value: $sliderValue) { isEditing in
    if isEditing {
        // dragging began
    } else {
        // dragging ended
    }
}
```

**Comment by AlexChekel1337** [comment]
The functionality was implemented in [v2.1.0](https://github.com/AlexChekel1337/SliderControl/releases/tag/v2.1.0) release
